pg_service_parser/metadata.txt

```diff
@@ -1,17 +1,18 @@
 [general]
-name=pg_service parser
+name=PG service parser
 description=Copy or override PG service entries
-about=Copy or override PG service entries for connection to PostgreSQL databases
-version=0.0.1
-qgisMinimumVersion=3.00
+about=Copy or override PG service entries for PostgreSQL connections
+version=dev
+qgisMinimumVersion=3.28
 qgisMaximumVersion=3.99
+supportsQt6=yes
```

Comment:
Looks good, but let's first merge this one https://github.com/opengisch/qgis-pg-service-parser-plugin/pull/2, so that I can start in experimental mode (first contact with Qt6).